### DISCUSSION TITLE
Resolve a conversation from within CSML bot

### DISCUSSION THREAD

**Discussion by hmeranda** [chatwoot/chatwoot#8133] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

Trying to resolve a conversation from within a CSML flow when using a bot script
for example, the bot already answered the consumer question, so there is no need to keep the conversation in status open as there nothing left for the agent to do

### Describe the solution you'd like

Some sort of exposed/injected method for setting the conversation status (making sure its not overwritten back to "open" when the CSML flow ends)

### Describe alternatives you've considered

Tried http posting to the toggleStatus endpoint by supplying the conversation id obtained from the CSML global variable _metadata.
it results with success and the new status is indeed "Resolved" but when the bot flow ends a line later it seems to change it back to "open"

### Additional context

Thank you very much :)

**Comment by pranavrajs** [comment]
@hmeranda Thanks for the suggestion. We are moving this to discussions as we won't immediately prioritize this feature.
